base/applications/regedit/lang/zh-HK.rc

```diff
@@ -0,0 +1,562 @@
+    AUTORADIOBUTTON "十進制(&D)", IDC_FORMAT_DEC, 141, 59, 60, 10
+    DEFPUSHBUTTON "確定", IDOK, 142, 82, 50, 14
+    PUSHBUTTON "取消", IDCANCEL, 196, 82, 50, 14
+END
+
+IDD_EDIT_RESOURCE_LIST DIALOGEX 32, 24, 170, 120
+STYLE DS_SHELLFONT | DS_MODALFRAME | DS_NOIDLEMSG | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+CAPTION "資源列表"
+FONT 9, "新細明體"
+BEGIN
+    CONTROL "", IDC_RESOURCE_LIST, "SysListView32", LVS_REPORT | LVS_NOSORTHEADER |
+            LVS_SINGLESEL | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 10, 10, 150, 75
+    DEFPUSHBUTTON "確定", IDOK, 27, 100, 50, 14
+    PUSHBUTTON "顯示...", IDC_SHOW_RESOURCE, 93, 100, 50, 14, WS_DISABLED
+END
+
+IDD_EDIT_RESOURCE DIALOGEX 6, 18, 281, 283
+STYLE DS_SHELLFONT | DS_MODALFRAME | DS_NOIDLEMSG | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+CAPTION "資源"
+FONT 9, "新細明體"
+BEGIN
+    LTEXT "DMA：", IDC_STATIC, 10, 5, 20, 8
+    CONTROL "", IDC_DMA_LIST, "SysListView32", LVS_REPORT | LVS_NOSORTHEADER |
+            LVS_SINGLESEL | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 10, 15, 261, 31
+    LTEXT "中斷：", IDC_STATIC, 10, 48, 35, 8
+    CONTROL "", IDC_IRQ_LIST, "SysListView32", LVS_REPORT | LVS_NOSORTHEADER |
+            LVS_SINGLESEL | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 10, 58, 261, 31
+    LTEXT "記憶體：", IDC_STATIC, 10, 91, 32, 8
+    CONTROL "", IDC_MEMORY_LIST, "SysListView32", LVS_REPORT | LVS_NOSORTHEADER |
+            LVS_SINGLESEL | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 10, 101, 261, 31
+    LTEXT "埠：", IDC_STATIC, 10, 134, 38, 8
+    CONTROL "", IDC_PORT_LIST, "SysListView32", LVS_REPORT | LVS_NOSORTHEADER |
+            LVS_SINGLESEL | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 10, 144, 261, 31
+    LTEXT "裝置特定的資料：", IDC_STATIC, 10, 177, 80, 8
+    CONTROL "", IDC_DEVICE_LIST, "SysListView32", LVS_REPORT | LVS_NOSORTHEADER |
+            LVS_SINGLESEL | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 10, 188, 261, 31
+    GROUPBOX "用法", IDC_STATIC, 10, 219, 125, 39
+    LTEXT "待定", IDC_UNDETERMINED, 18, 230, 46, 8, WS_DISABLED
+    LTEXT "共享", IDC_SHARED, 18, 240, 48, 8, WS_DISABLED
+    LTEXT "專用裝置", IDC_DEVICE_EXCLUSIVE, 72, 230, 58, 8, WS_DISABLED
+    LTEXT "專用驅動程式", IDC_DRIVER_EXCLUSIVE, 72, 240, 54, 8, WS_DISABLED
+    RTEXT "分介面類型：", IDC_STATIC, 144, 220, 58, 8
+    LTEXT "", IDC_INTERFACETYPE, 205, 220, 66, 8
+    RTEXT "匯流排編號：", IDC_STATIC, 151, 230, 51, 8
+    LTEXT "", IDC_BUSNUMBER, 205, 230, 66, 8
+    RTEXT "版本：", IDC_STATIC, 151, 240, 51, 8
+    LTEXT "", IDC_VERSION, 205, 240, 66, 8
+    RTEXT "修訂本：", IDC_STATIC, 151, 250, 51, 8
+    LTEXT "", IDC_REVISION, 205, 250, 66, 8
+    DEFPUSHBUTTON "確定", IDOK, 92, 263, 40, 14
+    PUSHBUTTON "資料(&D)...", IDC_SHOW_RESOURCE_DATA, 148, 263, 40, 14, WS_DISABLED
+END
+
+/* String Tables */
+
+STRINGTABLE
+BEGIN
+    IDS_LIST_COLUMN_NAME "名稱"
+    IDS_LIST_COLUMN_TYPE "類型"
+    IDS_LIST_COLUMN_DATA "資料"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_APP_TITLE "登錄編輯程式"
```

Comment:
same here

Reply:
This is also the actual Windows translation. Changing it into different terms can be weird and confusing.

Also in [wikipedia](https://zh.wikipedia.org/wiki/%E6%B3%A8%E5%86%8C%E8%A1%A8), it says that Taiwan, HK and Macau uses 
"登錄檔" instead of "註冊表".

I only post this response for all the "Registry" related problem here to reduce comment spam.